tests/google_tests/env.cpp

```diff
@@ -0,0 +1,65 @@
+#include <gtest/gtest.h>
+
+extern "C" {
+#include "env.h"
+}
+
+class EnvTest : public ::testing::Test {
+protected:
+	static void SetUpTestCase() {}
+	static void TearDownTestCase() {}
+
+	void SetUp() {
+	}
+	void TearDown() {
+	}
+};
+
+
+TEST_F(EnvTest, normal_1)
+{
+	set_env("testkey","testval");
```

Comment:
特に挙動が変わる訳ではないですが、それっぽい値を使いたいかも。
```suggestion
	set_env("HOME","/Users/takkatao");
```